api/ResponseTypes.d.ts

```diff
@@ -0,0 +1,132 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+interface SearchResponse<TSource = unknown, TAggregations = unknown> {
+  took: number
+  timed_out: boolean
+  _scroll_id?: string
+  _shards: Shards
+  hits: {
+    total: {
+      value: number
+      relation: string
```

Comment:
```suggestion
      relation: "eq", "gte"
```

Reply:
I'm surprised that we are able to generate request types from the API spec, but no responses. https://github.com/elastic/elasticsearch/blob/d7bbc9df1d3edf47f8d1c2f0983c88c64d78a4b4/rest-api-spec/src/main/resources/rest-api-spec/api/search.json#L231 
@delvedor Do you know if the elasticesearch team is going to introduce a spec for API responses as well?